src/internal/node_api_internal.h

```diff
@@ -19,6 +19,7 @@
 #include "iotjs_def.h"
 #include "jerryscript-ext/handle-scope.h"
 #include "jerryscript.h"
+#define NAPI_EXPERIMENTAL
```

Comment:
Where does this be used?

Reply:
It's used in `node_api.h`, in which declares napi experimental functions if `NAPI_EXPERIMENTAL` has been defined.